### ISSUE TITLE
Error when clicking on preferences

### ISSUE THREAD

**Issue by cornytrace** [odoo/odoo#69414]
Impacted versions:
Odoo 13.0-20210416

Steps to reproduce:
Click on the user in the top-right corner, and click on preferences.
Also happens when trying to make a new mass mailing.
EDIT: Also happens when clicking on "module info" in Apps

Current behavior:
A window pops up with "Odoo client error", sometimes it shows this:
```
Error:


Traceback:
Error: instance[methodName] is undefined
willStart/</<@http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:4314:66
_.forEach@http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:108:566
willStart/<@http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:4314:21
```
Other times it shows:
```
Traceback:
Error: instance[methodName] is undefined
willStart/</<@http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:4314:66
_.forEach@http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:108:566
willStart/<@http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:4314:21
promise callback*willStart@http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:4312:567
OdooClass.extend/</prototype[name]</<@http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:3861:488
attachTo@http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:4094:376
willStart/assetsLoaded<@http://192.168.2.4:8069/web/content/1098-2ea12fe/web.assets_backend.js:2601:103
willStart@http://192.168.2.4:8069/web/content/1098-2ea12fe/web.assets_backend.js:2601:32
OdooClass.extend/</prototype[name]</<@http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:3861:488
_widgetRenderAndInsert@http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:4103:153
_renderFieldWidget@http://192.168.2.4:8069/web/content/1098-2ea12fe/web.assets_backend.js:1416:80
_renderInnerGroupField@http://192.168.2.4:8069/web/content/1098-2ea12fe/web.assets_backend.js:1694:381
_renderInnerGroup/<@http://192.168.2.4:8069/web/content/1098-2ea12fe/web.assets_backend.js:1691:44
_renderInnerGroup@http://192.168.2.4:8069/web/content/1098-2ea12fe/web.assets_backend.js:1688:75
_renderTagGroup@http://192.168.2.4:8069/web/content/1098-2ea12fe/web.assets_backend.js:1703:248
_renderNode@http://192.168.2.4:8069/web/content/1098-2ea12fe/web.assets_backend.js:1695:246
_renderNode@http://192.168.2.4:8069/web/content/1098-2ea12fe/web.assets_backend.js:3105:569
OdooClass.extend/Class.include/</prototype[name]</<@http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:3864:371
_renderTagForm@http://192.168.2.4:8069/web/content/1098-2ea12fe/web.assets_backend.js:1703:28
_renderNode@http://192.168.2.4:8069/web/content/1098-2ea12fe/web.assets_backend.js:1695:246
_renderNode@http://192.168.2.4:8069/web/content/1098-2ea12fe/web.assets_backend.js:3105:569
OdooClass.extend/Class.include/</prototype[name]</<@http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:3864:371
_renderView@http://192.168.2.4:8069/web/content/1098-2ea12fe/web.assets_backend.js:1709:1133
_render@http://192.168.2.4:8069/web/content/1098-2ea12fe/web.assets_backend.js:1415:222
start@http://192.168.2.4:8069/web/content/1098-2ea12fe/web.assets_backend.js:1083:26
OdooClass.extend/</prototype[name]</<@http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:3861:488
start@http://192.168.2.4:8069/web/content/1098-2ea12fe/web.assets_backend.js:1675:20
OdooClass.extend/</prototype[name]</<@http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:3861:488
_widgetRenderAndInsert/<@http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:4104:52
promise callback*_widgetRenderAndInsert@http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:4103:165
appendTo@http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:4094:220
_startRenderer@http://192.168.2.4:8069/web/content/1098-2ea12fe/web.assets_backend.js:1109:72
start@http://192.168.2.4:8069/web/content/1098-2ea12fe/web.assets_backend.js:344:785
OdooClass.extend/</prototype[name]</<@http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:3861:488
start@http://192.168.2.4:8069/web/content/1098-2ea12fe/web.assets_backend.js:1091:59
OdooClass.extend/</prototype[name]</<@http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:3861:488
start@http://192.168.2.4:8069/web/content/1098-2ea12fe/web.assets_backend.js:1374:1262
OdooClass.extend/</prototype[name]</<@http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:3861:488
_widgetRenderAndInsert/<@http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:4104:52
promise callback*_widgetRenderAndInsert@http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:4103:165
appendTo@http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:4094:220
_startController@http://192.168.2.4:8069/web/content/1098-2ea12fe/web.assets_backend.js:432:325
_executeActionInDialog@http://192.168.2.4:8069/web/content/1098-2ea12fe/web.assets_backend.js:415:347
_executeAction@http://192.168.2.4:8069/web/content/1098-2ea12fe/web.assets_backend.js:412:768
_executeWindowAction/</<@http://192.168.2.4:8069/web/content/1098-2ea12fe/web.assets_backend.js:446:380
promise callback*_executeWindowAction/<@http://192.168.2.4:8069/web/content/1098-2ea12fe/web.assets_backend.js:446:306
promise callback*_executeWindowAction@http://192.168.2.4:8069/web/content/1098-2ea12fe/web.assets_backend.js:442:455
_handleAction@http://192.168.2.4:8069/web/content/1098-2ea12fe/web.assets_backend.js:448:109
OdooClass.extend/Class.include/</prototype[name]</<@http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:3864:371
_handleAction@http://192.168.2.4:8069/web/content/1098-2ea12fe/web.assets_backend.js:468:20
OdooClass.extend/Class.include/</prototype[name]</<@http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:3864:371
doAction/<@http://192.168.2.4:8069/web/content/1098-2ea12fe/web.assets_backend.js:410:185
promise callback*doAction@http://192.168.2.4:8069/web/content/1098-2ea12fe/web.assets_backend.js:410:42
do_action@http://192.168.2.4:8069/web/content/1098-2ea12fe/web.assets_backend.js:494:1291
do_action@http://192.168.2.4:8069/web/content/1098-2ea12fe/web.assets_backend.js:489:1849
proxy/<@http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:3946:11
trigger@http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:3944:180
_trigger_up@http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:3950:378
_trigger_up@http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:3950:462
_trigger_up@http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:3950:462
_trigger_up@http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:3950:462
trigger_up@http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:3950:280
do_action/<@http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:3994:1233
do_action@http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:3994:1191
callback/<@http://192.168.2.4:8069/web/content/1098-2ea12fe/web.assets_backend.js:779:945
promise callback*callback@http://192.168.2.4:8069/web/content/1098-2ea12fe/web.assets_backend.js:779:892
promise callback*clear_uncommitted_changes@http://192.168.2.4:8069/web/content/1098-2ea12fe/web.assets_backend.js:489:1033
proxy/<@http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:3946:11
trigger@http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:3944:180
_trigger_up@http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:3950:378
_trigger_up@http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:3950:462
_trigger_up@http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:3950:462
_trigger_up@http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:3950:462
trigger_up@http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:3950:280
_onMenuSettings@http://192.168.2.4:8069/web/content/1098-2ea12fe/web.assets_backend.js:779:781
start/<@http://192.168.2.4:8069/web/content/1098-2ea12fe/web.assets_backend.js:777:524
dispatch@http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:1024:447
add/elemData.handle@http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:1010:166
```
Expected behavior:
The preferences window opens

Video/Screenshot link (optional):

Support ticket number submitted via odoo.com/help (optional):

**Comment by pedrobaeza** [comment]
Hi, thanks for reporting. Are you able to reproduce the problem in runbot infrastructure (http://runbot.odoo.com/runbot - click on the blue door icon of the version row - admin/admin)?

If so, please explain with all the possible detailed steps to get to the error. Not having them, it's very difficult to hunt down the problem. Please don't be shy of adding details, and even better if you provide some screenshots or videos.

If not, this can be due to a configuration change, a custom/community module you have deployed in your DB or an outdated code. Please report back if it's the case.

For keeping a clean house in this issue tracker, if you don't answer in several days, this will be closed assuming it was one of the stated things.

**Comment by cornytrace** [comment]
I have been able to track down the error in the source code to the following line: https://github.com/odoo/odoo/blob/cd9c071c9357cef14635ef094a9f14fc5431956c/addons/web_editor/static/src/js/wysiwyg/root.js#L53

Furthermore, the above error was on Firefox. On Chrome the error is as follows:
```
Traceback:
TypeError: Cannot read property 'bind' of undefined
    at http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:4314:104
    at Function._.each._.forEach (http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:108:558)
    at http://192.168.2.4:8069/web/content/975-668acd9/web.assets_common.js:4314:21
```
There are currently no third party modules installed.

**Comment by pedrobaeza** [comment]
Are you able to reproduce it on runbot?

**Comment by cornytrace** [comment]
I was recreating things I did on my local install on runbot. On both, when I removed the "iap" module the following error appears:
```
Something went wrong !

Record does not exist or has been deleted.
(Record: ir.model.fields(7004,), User: 1)
```
I narrowed the error down to the uninstall of module "sms", which was being uninstalled because it depends on "iap". To make the uninstall work I had to manually delete all references to the database entry mentioned in the error message, and the uninstall succeeded. I cannot do this on runbot as I do not have direct database access. So I cannot say for certain if this issue has anything to do with it.
On runbot I also cannot uninstall "eCommerce" without a similar error:
```
Something went wrong !

Record does not exist or has been deleted.
(Record: ir.model.fields(16083,), User: 1)
```
Unfortunately due to these issues I cannot make the runbot instance more similar to my own. This is my module list: https://www.dropbox.com/s/hoztwcyil45cdjh/ir.module.module.xlsx?dl=0
